### ISSUE TITLE
Segfault djvu double click

### ISSUE THREAD

**Issue by Yanpas** [linuxmint/xreader#137]
STR: open djvu and double click somewhere in document
```
(gdb) bt
#0  0x00007f6075aadc10 in cairo_surface_set_device_scale ()
   from /usr/lib/x86_64-linux-gnu/libcairo.so.2
#1  0x00007f6076f5bcb8 in ev_pixbuf_cache_get_selection_surface ()
   from /usr/lib/x86_64-linux-gnu/libxreaderview.so.3
#2  0x00007f6076f6ad1c in ?? ()
   from /usr/lib/x86_64-linux-gnu/libxreaderview.so.3
#3  0x00007f6076f6b2f7 in ?? ()
   from /usr/lib/x86_64-linux-gnu/libxreaderview.so.3
#4  0x00007f6076f6bebb in ?? ()
   from /usr/lib/x86_64-linux-gnu/libxreaderview.so.3
#5  0x00007f607646709a in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#6  0x00007f6074cca1d4 in ?? ()
   from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#7  0x00007f6074ce44b8 in g_signal_emit_valist ()
   from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#8  0x00007f6074ce508f in g_signal_emit ()
   from /usr/lib/x86_64-linux-gnu/libgobject-2.0.so.0
#9  0x00007f60765a4c3c in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#10 0x00007f60764643be in ?? () from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#11 0x00007f60764661bc in gtk_main_do_event ()
   from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#12 0x00007f6075fd3d92 in ?? () from /usr/lib/x86_64-linux-gnu/libgdk-3.so.0
#13 0x00007f60749f3197 in g_main_context_dispatch ()
---Type <return> to continue, or q <return> to quit---
   from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#14 0x00007f60749f33f0 in ?? () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#15 0x00007f60749f3712 in g_main_loop_run ()
   from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#16 0x00007f6076465395 in gtk_main ()
   from /usr/lib/x86_64-linux-gnu/libgtk-3.so.0
#17 0x000000000041cc81 in ?? ()
#18 0x00007f60740da830 in __libc_start_main (main=0x41c890, argc=2, 
    argv=0x7ffcf7ae2268, init=<optimized out>, fini=<optimized out>, 
    rtld_fini=<optimized out>, stack_end=0x7ffcf7ae2258)
    at ../csu/libc-start.c:291
```

**Comment by JosephMcc** [comment]
I grabbed this test file: https://en.wikisource.org/wiki/File:Test_rs20846.djvu

I couldn't reproduce this problem on 1.4.2.

**Comment by Yanpas** [comment]
I can, but I'm using xreader 1.2.2 from mint 18.1

**Comment by Yanpas** [comment]
It is already fixed by 19f6fcf97a41db1ef2645299de49eeb92a41d58c

Duplicate #92
